contrib/core-contract-tests/tests/pox-4/pox-4.stateful-prop.test.ts

```diff
@@ -87,6 +87,7 @@ it("statefully interacts with PoX-4", async () => {
       ustxBalance: initialUstxBalance,
       isStacking: false,
       hasDelegated: false,
+      hasLocked: [],
```

Comment:
@friedger @BowTiedRadone is this part of some domain language? Usually has/can are associated with Boolean fields. Could we name this as `lockedAddresses`, or `addressesLocked` to be consistent with the other fields (`amountLocked`, `amountUnlocked`)?

Reply:
:+1: This is historical :-) and should be changed

Reply:
I will rename to `lockedAddresses` 👍🏻 
Thank you!